### DISCUSSION TITLE
no instructions on how to run this

### DISCUSSION THREAD

**Discussion by Sturmey66** [librenms/docker#374] [Q&A]
everyone seems to believe that the people trying to run this are already experts at running things in docker. many of us are actually not developers and would like a straight forward explanation on how to use this. 

is there a way I can run this and actually have librenms usable on my system?

**Comment by crazy-max** [comment]
@Sturmey66 https://github.com/librenms/docker/blob/master/doc/usage.md

**Comment by Sturmey66** [comment]
so the this needs a database as well? that doesn't make this easier or better than installing bare metal.

**Comment by crazy-max** [comment]
You have everything in [one compose file](https://github.com/librenms/docker/blob/master/examples/compose/docker-compose.yml). Just spin up `docker-compose up -d` as explained in [usage doc](https://github.com/librenms/docker/blob/master/doc/usage.md).

**Comment by crazy-max** [comment]
Also this reference usage guides users through the setup based on `docker-compose`, but the installation of `docker-compose` and basic usage of Docker is out of scope of this documentation. To install `docker-compose` itself, follow the official [install instructions](https://docs.docker.com/compose/install/) and to get started with Docker follow the [official documentation](https://docs.docker.com/).
